test/network.spec.js

```diff
@@ -56,6 +56,7 @@ module.exports.addTests = function({testRunner, expect}) {
       expect(responses[0].fromCache()).toBe(false);
       expect(responses[0].fromServiceWorker()).toBe(false);
       expect(responses[0].request()).toBeTruthy();
+      expect(responses[0].remoteAddress()).toBe(`[::1]:${server.PORT}`);
```

Comment:
I'd try navigating to IP explicitly in line 51 to make this less flaky across environments (different setups might resolve
`localhost` differently)

```js
await page.goto(`http://[::1]:${server.PORT}/empty.html`);
```

Reply:
We'll still have trouble on IPv4-only machines as they won't be able to handle `[::1]`.  If we use `127.0.0.1` instead, Chrome still returns `[::1]` on IPv6-capable machines.